nitransforms/io/lta.py

```diff
@@ -191,6 +191,13 @@ def from_string(klass, string):
         lta = klass()
         sa = lta.structarr
         lines = string.splitlines()
+        while lines:
+            if not lines[0].startswith('type'):
+                lines.pop(0)
+                continue
+            break
+        if not lines:
+            raise Exception("Invalid LTA format")
```

Comment:
```suggestion
            raise TransformFileError("Invalid LTA format")
```
(and please add import on top)